bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -134,6 +134,7 @@ spec:
     Example MachineSet for other cloud providers:
     - [AWS](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md)
     - [Azure](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-azure.md)
+    - [GCP](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-gcp.md)
```

Comment:
The consensus in https://github.com/openshift/windows-machine-config-operator/pull/1232 was to point to official OCP docs over github. Does it make sense to wait for GCP support to be published and point there?

Reply:
why not add support for gcp in all places like README and supported platforms if we are adding it to csv?

Reply:
The consensus in #1232 was to NOT point to the official OCP docs over GitHub. We should update the docs only when releasing the community operator with GCP support.